src/main/kotlin/com/hedvig/underwriter/serviceIntegration/productPricing/ProductPricingServiceImpl.kt

```
@@ -131,5 +131,9 @@ class ProductPricingServiceImpl @Autowired constructor(
         productPricingClient.validateCampaign(campaignCode, country)
 
     override fun getContractsByMemberId(memberId: String): List<Contract> =
+        try {
         productPricingClient.getContractsByMemberId(memberId).body!!
+        } catch (exception: Exception) {
+            emptyList()
+        }
```

Feels a little bit dangerous. If e.g. PP is down we get that member has no contracts and takes business decisions on that which is wrong... Better to throw an exception and bail out...

---

yess very good point!!